libethereum/Client.cpp

```diff
@@ -930,3 +930,21 @@ ExecutionResult Client::call(Address const& _from, u256 _value, Address _dest, b
     }
     return ret;
 }
+
+std::tuple<h256, h256, h256> Client::getWork()
+{
+    bool oldShould = shouldServeWork();
```

Comment:
The comment got lost
```suggestion
    // lock the work so a later submission isn't invalidated by processing a transaction elsewhere.	
    // this will be reset as soon as a new block arrives, allowing more transactions to be processed.
    bool oldShould = shouldServeWork();
```